docs/build-on-polkadot/xcm/asset-transfer-api.md

```diff
@@ -0,0 +1,491 @@
+			export type LocalTransferTypes =
+				| 'assets::transfer'
+				| 'assets::transferKeepAlive'
+				| 'foreignAssets::transfer'
+				| 'foreignAssets::transferKeepAlive'
+				| 'balances::transfer'
+				| 'balances::transferKeepAlive'
+				| 'poolAssets::transfer'
+				| 'poolAssets::transferKeepAlive'
+				| 'tokens::transfer'
+				| 'tokens::transferKeepAlive';
+			```
+
+- `tx` ++"ConstructedFormat<T>"++ - The constructed transaction
+
+	??? code "Type `ConstructedFormat<T>`"
+
+		```javascript
+		export type ConstructedFormat<T> = T extends 'payload'
+		? GenericExtrinsicPayload
+		: T extends 'call'
+		? `0x${string}`
+		: T extends 'submittable'
+			? SubmittableExtrinsic<'promise', ISubmittableResult>
+			: never;
+		```
+
+		The `ConstructedFormat` type is a conditional type that returns a specific type based on the value of the TxResult `format` field.
+
+		- Payload Format - if the format field is set to 'payload', the ConstructedFormat type will return a [GenericExtrinsicPayload](https://github.com/polkadot-js/api/blob/3b7b44f048ff515579dd233ea6964acec39c0589/packages/types/src/extrinsic/ExtrinsicPayload.ts#L48)
+		- Call Format - if the format field is set to 'call', the ConstructedFormat type will return a hexadecimal string (0x${string}). This is the encoded representation of the extrinsic call
+		- Submittable Format - if the format field is set to 'submittable', the ConstructedFormat type will return a [SubmittableExtrinsic](https://github.com/polkadot-js/api/blob/3b7b44f048ff515579dd233ea6964acec39c0589/packages/api-base/src/types/submittable.ts#L56). This is a Polkadot-JS type that represents a transaction that can be submitted to the blockchain
+
+## Examples
+
+### Relay to System Parachain
+
+This example demonstrates how to initiate a cross-chain token transfer from a relay chain to a system parachain. Specifically, 1 WND will be transfered from a Westend (Relay Chain) account to a Westmint (System Parachain) account.
+
+```javascript
+import {
+  AssetTransferApi,
+  constructApiPromise,
+} from "@substrate/asset-transfer-api";
+
+async function main() {
+  const { api, specName, safeXcmVersion } = await constructApiPromise(
+    "wss://westend-rpc.polkadot.io"
+  );
+  const assetApi = new AssetTransferApi(api, specName, safeXcmVersion);
+  let callInfo;
+  try {
+    callInfo = await assetApi.createTransferTransaction(
+      "1000",
+      "5EWNeodpcQ6iYibJ3jmWVe85nsok1EDG8Kk3aFg8ZzpfY1qX",
+      ["WND"],
+      ["1000000000000"],
+      {
+        format: "call",
+        xcmVersion: safeXcmVersion,
+      }
+    );
+
+    console.log(`Call data:\n${JSON.stringify(callInfo, null, 4)}`);
+  } catch (e) {
+    console.error(e);
+    throw Error(e as string);
+  }
+
+  const decoded = assetApi.decodeExtrinsic(callInfo.tx, "call");
+  console.log(
+    `\nDecoded tx:\n ${JSON.stringify(JSON.parse(decoded), null, 4)}`
+  );
+}
+
+main()
+  .catch((err) => console.error(err))
+  .finally(() => process.exit());
+```
+
```

Comment:
Maybe it's worth adding a small sentence to introduce the following terminal output, such as: "The script execution will prompt the following on your terminal..."